src/Http/Http/perf/Microbenchmarks/GlobalUsings.cs

```diff
@@ -0,0 +1,3 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+global using Xunit;
```

Comment:
We need this because the shared `RequestDelegateCreationTestBase` has some asserts. I'm just importing Xunit into the context instead of adding if-defs to the shared code to avoid calling `Assert` all the time.

Reply:
NIT: Can we add this explanation as a comment to this line 🙂